src/components/Certify/Certify.vue

```diff
@@ -159,6 +166,10 @@ export default class Certify extends Vue {
   color: $gray7;
   font-weight: normal;
 }
+.firstCol, .secondCol {
+    padding-left: 0px;
+    padding-right: 0px;
```

Comment:
I have found that it's often easier to have these layout tweaks right in the template (HTML) -- especially when one has to update a bunch of layout on a page. In this case you can use the built-in Vuetify classes to set the padding: `px-0`.

Reply:
I made this change